src/store/models/Post.js

```diff
@@ -8,7 +8,7 @@ export const PROPOSAL_ADVICE = 'Advice'
 export const PROPOSAL_CONSENT = 'Consent'
 export const PROPOSAL_CONSENSUS = 'Consensus'
 export const PROPOSAL_SCHEDULING = 'Scheduling'
-export const PROPOSAL_GRADIENT = 'Gradient of Agreement'
+export const PROPOSAL_GRADIENT = 'Agree/Disagree'
```

Comment:
I actually prefer Gradient of Agreement since it's a long standing methodology for many decades. so... 🤷‍♂️